src/Dapr.Actors/Runtime/ActorConfig.cs

```diff
@@ -0,0 +1,116 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors.Runtime
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Represents Dapr actor configuration for this app.  See
+    /// https://github.com/dapr/docs/blob/master/reference/api/actors_api.md
+    /// </summary>
+    public sealed class ActorConfig
+    {
+        /// <summary>
+        /// Default constructor.  This is only here for the unit test.
+        /// </summary>
+        private ActorConfig()
+            : this(null, null, null, false)
+        { }
+
+        /// <summary>
+        /// Users should use this constructor to override default values.
+        /// </summary>
+        /// <param name="actorIdleTimeout"></param>
+        /// <param name="actorScanInterval"></param>
+        /// <param name="drainOngoingCallTimeout"></param>
+        /// <param name="drainBalancedActors"></param>
+        public ActorConfig(
+            TimeSpan? actorIdleTimeout = null,
+            TimeSpan? actorScanInterval = null,
+            TimeSpan? drainOngoingCallTimeout = default,
+            bool drainBalancedActors = default)
+        {
+            this.RegisteredActorTypes = new List<string>();
```

Comment:
Registered ActorTypes should stay in ActorRuntime and should be exposed in ActorConfig